### DISCUSSION TITLE
Backup method?

### DISCUSSION THREAD

**Discussion by jhmonroe** [danny-avila/LibreChat#5005] [Feature Requests & Suggestions]
1) Any plans to soon add a setting to export chats and also save other data like agents we've created?
2) What is your recommended way to backup our chats/agents/etc?

I tried to connect with MongoDB Compass and not able to get it to connect.
Tried the username/password from yaml file.
Tried various combinations of the local port Docker/LibreChat set it up in. localhost:3080.... 27017... 

(When I get a new Mac I'll install everything fresh to declutter my old drive instead of moving everything over as-is... 😄 )

**Comment by budu** [comment]
There's another discussion here: https://github.com/danny-avila/LibreChat/discussions/3136
